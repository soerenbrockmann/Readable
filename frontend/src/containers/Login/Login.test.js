import React from "react";

describe("<Login />", () => {
  const login = jest.mock("./Login", () => "Login");
  it("renders properly", () => {
    expect(login).toMatchSnapshot();
  });
});
