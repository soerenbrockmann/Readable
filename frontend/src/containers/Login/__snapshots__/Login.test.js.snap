// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login /> renders properly 1`] = `
Object {
  "addMatchers": [Function],
  "autoMockOff": [Function],
  "autoMockOn": [Function],
  "clearAllMocks": [Function],
  "clearAllTimers": [Function],
  "deepUnmock": [Function],
  "disableAutomock": [Function],
  "doMock": [Function],
  "dontMock": [Function],
  "enableAutomock": [Function],
  "fn": [Function],
  "genMockFn": [Function],
  "genMockFromModule": [Function],
  "genMockFunction": [Function],
  "isMockFunction": [Function],
  "mock": [Function],
  "resetAllMocks": [Function],
  "resetModuleRegistry": [Function],
  "resetModules": [Function],
  "runAllImmediates": [Function],
  "runAllTicks": [Function],
  "runAllTimers": [Function],
  "runOnlyPendingTimers": [Function],
  "runTimersToTime": [Function],
  "setMock": [Function],
  "spyOn": [Function],
  "unmock": [Function],
  "useFakeTimers": [Function],
  "useRealTimers": [Function],
}
`;
